### PR TITLE
[#16] Implement ToRow instance for NonEmpty

### LINKED ISSUE #16: ToRow instance for `NonEmpty`
It would be very helpful.

### PR DESCRIPTION
**Pull request by vrom911**
Resolves #16

### CODE REVIEW

File: src/Database/PostgreSQL/Simple/ToRow.hs [hunk 1]
```diff
@@ -21,6 +21,8 @@ module Database.PostgreSQL.Simple.ToRow
       ToRow(..)
     ) where
 
+import Data.Foldable (toList)
```
Comment (phadej) [inline src/Database/PostgreSQL/Simple/ToRow.hs]: I'd prefer `import qualified Data.List.NonEmpty as NE` and `NE.toList`. `toList` become a method of `Foldable` quite recently, so it doesn't use efficient impl on older GHCs.

File: src/Database/PostgreSQL/Simple/ToRow.hs [hunk 2]
```diff
@@ -198,6 +200,9 @@ instance (ToField a, ToField b, ToField c, ToField d, ToField e, ToField f,
 instance (ToField a) => ToRow [a] where
     toRow = map toField
 
+instance (ToField a) => ToRow (NonEmpty a) where
```
Comment (phadej) [inline src/Database/PostgreSQL/Simple/ToRow.hs]: Please add corresponding `FromRow` instance as well.

Comment (vrom911) [reply src/Database/PostgreSQL/Simple/ToRow.hs]: `FromRow` is out of the scope of this issue. I don't mind if one would implement this as the separate issue/PR.

Comment (phadej) [reply src/Database/PostgreSQL/Simple/ToRow.hs]: I'm sorry, but I won't accept a PR without symmetry.

Comment (vrom911) [reply src/Database/PostgreSQL/Simple/ToRow.hs]: Then you should have told that in the issue (which clearly states what is needed to be implemented) and you don't waste your time.

**Pull request closed without merging**